orchagent/fdborch.cpp

```diff
@@ -300,6 +389,47 @@ void FdbOrch::update(sai_fdb_event_t        type,
             }
         }
 
+        // If MAC is MCLAG remote do not delete for age event, Add the MAC back..
+        if (existing_entry->second.origin == FDB_ORIGIN_MCLAG_ADVERTIZED)
```

Comment:
i think this can't be reach since addFdbEntry() will modify origin to FDB_ORIGIN_LEARN to store.

Reply:
@pettershao-ragilenetworks : This is to address timing scenarios and platforms which doesnt support SAI_FDB_ENTRY_ATTR_ALLOW_MAC_MOVE. If the MAC is added by ICCPD then the origin is set as FDB_ORIGIN_MCLAG (not to age) If the FdbOrch receives an AGE event ( in timing scenarios or on the platform which doesnt have support for SAI_FDB_ENTRY_ATTR_ALLOW_MAC_MOVE)  and MAC entry origin is FDB_ORIGIN_MCLAG, since the MAC is remotely leared it should not get deleted so FdbOrch re-adds the MAC entry.